docs/02_cleos/03_command-reference/net/disconnect.md

```diff
@@ -1,17 +1,56 @@
+## Command
+```sh
+cleos net disconnect [OPTIONS] host
+```
+
+**Where:**
+* [OPTIONS] = See **Options** in the [**Command Usage**](command-usage) section below.
+* host = The hostname:port to disconnect from.
+
+**Note:** The arguments and options enclosed in square brackets are optional.
+
 ## Description
-close an existing connection
+Close an existing connection to a specified peer. This command allows an operator's node to disconnect from another peer without restarting the node.
```

Comment:
Agin should this be Use this command to ....

Reply:
Updated to: "A node operator can use this command..."